backend/core/database.py

```diff
@@ -858,9 +858,11 @@ def handle_hijack_ongoing_request(self, message):
                 try:
                     results = []
                     query = (
-                        "SELECT DISTINCT ON(h.key) b.key, b.prefix, b.as_path, b.type, h.key, h.hijack_as, h.type "
+                        "SELECT b.key, b.prefix, b.origin_as, b.as_path, b.type, b.peer_asn, "
+                        "b.communities, b.timestamp, b.service, b.matched_prefix, h.key, h.hijack_as, "
+                        "h.type, h.time_detected "
                         "FROM hijacks AS h LEFT JOIN bgp_updates AS b ON (h.key = ANY(b.hijack_key)) "
-                        "WHERE h.active = true AND b.type='A' AND b.handled=true"
+                        "WHERE h.active = true AND b.handled=true"
```

Comment:
I also send withdrawals since they are important to replicate the old outdated bug as a correct active bug with different type and properties.